spec/abstractops.html

```diff
@@ -1433,7 +1433,7 @@ <h1>ParseTemporalTimeZoneString ( _isoString_ )</h1>
         1. Set _seconds_ to ! ToIntegerOrInfinity(_seconds_).
         1. If _fraction_ is not *undefined*, then
           1. Set _fraction_ to the string-concatenation of the previous value of _fraction_ and the string *"000000000"*.
-          1. Let _nanoseconds_ be the String value equal to the substring of _fraction_ from 0 to 9.
+          1. Let _nanoseconds_ be the String value equal to the substring of _fraction_ consisting of the code units with indices 1 (inclusive) through 10 (exclusive).
```

Comment:
i think "the substring of" already has a defined meaning wrt the bounds: https://tc39.es/ecma262/#substring so i'm not sure you need to specify that (iow, the exact phrase the spec requires is `the substring of _fraction_ from 1 to 10`.

Reply:
ok change it back